includes/batch.inc

```diff
@@ -250,6 +265,38 @@ function islandora_paged_content_create_ocr_batch_operation(array $pages, array
 }
 
 /**
+ * Batch operation for consolidating OCR.
+ *
+ * @param string $paged_content_id
+ *   The paged content identifier to store the OCR.
+ * @param array $context
+ *   The context of this batch operation.
+ */
+function islandora_paged_content_create_ocr_paged_content_batch_operation($paged_content_id, &$context) {
+  module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
+  $paged_content = islandora_object_load($paged_content_id);
+  if (empty($context['results']['pages'])) {
+    return;
+  }
+  islandora_paged_content_start_batch_operation($context['results']['pages'], $context);
+  // Opting to write the consolidated OCR to the filesystem to avoid issues with
+  // PHP's memory limit on massive books.
+  $consolidated_ocr = drupal_realpath("temporary://{$paged_content->id}_OCR.txt");
```

Comment:
Also, thinking this should probably be registered as a managed temporary file, just in case something bad happens.